Assets/RenderGraphs/DEFERRED_PBR.lrg

```diff
@@ -307,8 +307,8 @@
                 "texture_type": "TEXTURE_CUBE",
                 "x_dim_type": "CONSTANT",
                 "y_dim_type": "CONSTANT",
-                "x_dim_var": 512.0,
-                "y_dim_var": 512.0,
+                "x_dim_var": 1024.0,
+                "y_dim_var": 1024.0,
```

Comment:
Is it necessary to have increased size pushed to master?

Reply:
Yes looks very nice 👍

Reply:
Ok very good 👍